bps.go

```diff
@@ -3,6 +3,9 @@ package bps
 import "github.com/google/uuid"
 
 // GenClientID generates random client ID.
+// It is guaranteed to:
+// only start with [a-z],
+// only contain [0-9A-Za-z_-]
```

Comment:
no `_` possible

Reply:
I thought I'll specify it for future, but let's remove.